dist/backdraft.js

```diff
@@ -532,6 +532,16 @@ _.extend(Plugin.factory, {
       return !columnConfig.bulk;
     });
 
+    // read initial filters from URL
+    var urlParamString = window.location.href.split("?")[1];
+    if (urlParamString && $.deparam(urlParamString) && $.deparam(urlParamString).ext_filter_json ) {
```

Comment:
I would vote for implementing it while we are here, seems like QA has a bit of work to get through at the moment so I think that gives us some time. If we end up waiting on this story at any point then that might be a good idea